Core/Core.Platform.Android/Storage/StorageManager.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using Android.Content;
+using Java.Lang;
+
+namespace AeroGear.Mobile.Core.Storage
+{
+	public class StorageManager : IStorageManager
```

Comment:
should we register this in `DependencyService`? see example: https://github.com/aerogear/aerogear-xamarin-sdk/blob/master/Core/Core.Platform.Android/AndroidPlatformInjector.cs#L6

Reply:
@aidenkeating Based on the latest comment from @wojta, I don't think we need to worry about the `DependencyService` for now.